yastd/lib/yast2/dbus/service.rb

```diff
@@ -38,7 +38,7 @@ class Service
       SERVICE_NAME = "org.opensuse.YaST"
 
       # @return [String] D-Bus object path
-      OBJECT_PATH = "/org/opensuse/YaST/Installer"
+      OBJECT_PATH = "/org/opensuse/YaST/Installer1"
```

Comment:
Now that we are using separate branches, you could use the same path. It is true that perhaps we could run both APIs in parallel (I do not know), but I do not think it is a valid use case by now.

Reply:
well, this is intentional change. It is dbus convention to always include version number in object path...so you can provide multiple versions on dbus. BTW I think it would be nicer to have `/org/opensuse/DInstaller1` and we should probably also change service name.

Reply:
Oh, I did not know that. For instance, DBus, NetworkManager, PackageKit or UPower do not follow that approach; but, on the other hand, FirewallD, UDisks or PolicyKit do it.

Interesting topic. And yes, we could think about renaming the service and the path as you propose.